sys/cheri/c18n.h

```diff
@@ -59,6 +59,15 @@ struct rtld_c18n_stats {
 	_Atomic(size_t) rcs_switch;
 };
 
+/*
+ * Compartment information exposed by RTLD.
+ */
+struct rtld_c18n_compart {
+	const char *	rcc_name;
```

Comment:
I had to add an `__kerncap` here...

Reply:
Fixed now. Not sure why CI is now happy with the lack of __kerncap...